scrapers/Nubiles.yml

```diff
@@ -12,6 +12,7 @@ sceneByURL:
       - badteenspunished.com/video/watch/
       - bountyhunterporn.com/video/watch/
       - brattymilf.com/video/watch/
+      - caughtmycoach.com/tube/watch/
```

Comment:
I can't find any video links on their sites that use the `/tube/watch` pattern, they're all `/video/watch` as far as I can tell... normally I'd just merge and replace this, but considering Chesterton's Fence I thought I'd ask about and get some context first 🙂